hypha/apply/projects/migrations/0080_auto_20231006_1757.py

```diff
@@ -0,0 +1,1423 @@
+                                    group="Fields",
+                                ),
+                            ),
+                            (
+                                "markdown_text",
+                                wagtail.blocks.StructBlock(
+                                    [
+                                        (
+                                            "field_label",
+                                            wagtail.blocks.CharBlock(label="Label"),
+                                        ),
+                                        (
+                                            "help_text",
+                                            wagtail.blocks.TextBlock(
+                                                label="Help text", required=False
+                                            ),
+                                        ),
+                                        (
+                                            "help_link",
+                                            wagtail.blocks.URLBlock(
+                                                label="Help link", required=False
+                                            ),
+                                        ),
+                                        (
+                                            "required",
+                                            wagtail.blocks.BooleanBlock(
+                                                label="Required", required=False
+                                            ),
+                                        ),
+                                        (
+                                            "default_value",
+                                            wagtail.blocks.TextBlock(
+                                                label="Default value", required=False
+                                            ),
+                                        ),
+                                        (
+                                            "word_limit",
+                                            wagtail.blocks.IntegerBlock(
+                                                default=1000, label="Word limit"
+                                            ),
+                                        ),
+                                    ],
+                                    group="Fields",
+                                ),
+                            ),
+                        ],
+                        use_json_field=True,
+                    ),
+                ),
+            ],
+            options={
+                "abstract": False,
+            },
+            bases=(hypha.apply.stream_forms.models.BaseStreamForm, models.Model),
+        ),
+        migrations.RemoveField(
+            model_name="reportversion",
+            name="private_content",
+        ),
+        migrations.RemoveField(
+            model_name="reportversion",
+            name="public_content",
+        ),
```

Comment:
This probably wants to be two (or three) migrations rather than one, eventually.

Reply:
We actually ended up squashing it into one.